confidant/models/credential.py

```diff
@@ -93,6 +101,17 @@ def diff(self, other_cred):
                 'added': new.documentation,
                 'removed': old.documentation
             }
+        if set(old.tags) != set(new.tags):
+            diff['tags'] = {
+                'added': new.tags,
+                'removed': old.tags
+            }
```

Comment:
For this you need to do a list diff for added/removed. added should only contain fields that were actually added, and removed should only contain fields actually removed. We only display the diff for the field if things changed.

Reply:
Yea good point